frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/OldFeatureStrategyMenu.tsx

```diff
@@ -0,0 +1,254 @@
+import type React from 'react';
```

Comment:
Maybe consider adding a comment here so that we remember to delete it when we clean up the flag?

```suggestion
// deprecated; delete with flag `newStrategyDropdown`
import type React from 'react';
```